app/models.py

```diff
@@ -0,0 +1,179 @@
+from tinydb import TinyDB, where, Query
+import os
+import io
+import csv
+from app.data import CSV_FIELDS, FIELD_MAPPING, LABELS
+
+this_dir = os.path.dirname(__file__)
+db = TinyDB(os.path.join(this_dir, '../data/db.json'))
+
+# These functions are meant to be the only ones that access the db
+# directly. If we ever decide to migrate from tinydb, that can all be
+# coordinated here.
+
+# Data loads should clear the entire database first.
+def clear_database():
+  db.purge_tables()
```

Comment:
The data loading script completely purges and reloads the database from scratch.